suites/squid/nvmeof/tier-2_8-1_auto-load-balancing_tests.yaml

```diff
@@ -0,0 +1,282 @@
+# test suite for auto load balancing feature
+# cluster configuration file: conf/squid/nvmeof/ceph_nvmeof_4-nvmeof-gwgroup_2gw_cluster.yaml
+# inventory: conf/inventory/rhel-9.3-server-x86_64-xlarge.yaml
+
+tests:
```

Comment:
Do we need a new suite file for 8.1? Can't we use the existing suite for loadbalancing?

Reply:
@HaruChebrolu Initially I went with same approach but had to write new because of below reasons

-  this is a new feature where tests steps especially on validation vary from 8.0 and hence new polarion IDs and hence new suite because of new feature
-  Auto load balancing is best to verify with more number of namespaces
-  There are more scenarios like namespace deletion and namespaces addition to same LB group and deletion from same LB group that will be added to this suite  which are applicable to only 8.1 and irrelevant to 8.0